_posts/2018-01-22-threads.markdown

```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+author: Santiago Doldán
+title: Introduce threads to Suttna
+image: posts/threads/threads.png
+categories: slack teams threads bot suttna
+---
+
+![App](/assets/posts/threads/threads.png)
+
+# We are really proud to announce that Suttna now supports threads 😌
+
+When we started building Suttna, our main goal was to let users configure everything without leaving their messaging client.
+
+
+Having that in mind, our first idea was to create a subset of dialogs where users would be able to just start a conversation running
+a command (e.g. `@suttna configure`) and Suttna will guide them according to what they wanted to do.
+
+<br />
+
+![Excelent Idea](/assets/posts/threads/excelent-idea.jpg)
+
+<br />
+
+But we forgot something very important ... What happens with ALL the other users that are part of the channel but are not talking with Suttna? they might be asking
```

Comment:
capaz que en vez de "nos olvidamos algo muy importante", encararlo más por, lo logramos, pero todavía había algo que podíamos mejorar.

Reply:
Que opinan de cambiarlo a algo como:

`One of the aspects that we didn't thought at first was, how much distraction Suttna can create on a particular channel ? How can we avoid distracting team members ?`